### DISCUSSION TITLE
Can not reconnect

### DISCUSSION THREAD

**Discussion by HiSunzhenliang** [Eugeny/tabby#6554] [Q&A] [answered]
**Does anyone encounter the problem of being unable to reconnect after disconnection ？**
![image](https://user-images.githubusercontent.com/42871690/172106015-cbd4bc20-5821-48f1-96c9-01937f06228c.png)

**Comment by Fu1249519541** [comment]
过一段时间不能连接，只能关闭重新打开 
报错：SSH   X  Remote rejected opening a shell channel: Error: (SSH) Channel open failure: open failed

**Reply by Hans-zhao831** [reply]
> 过一段时间不能连接，只能关闭重新打开 报错：SSH X Remote rejected opening a shell channel: Error: (SSH) Channel open failure: open failed

我是一直不能重新连接。解决办法是关了重新开。这个bug真的是好烦。PS.我的是apple M1的

**Comment by ouyangxudu** [comment]
me too.
<img width="587" alt="image" src="https://user-images.githubusercontent.com/22606068/195777535-c432dace-a7ce-4878-bed9-67033a3f6b3a.png">

**Comment by HiSunzhenliang** [comment, marked as answer]
Fixed. https://github.com/Eugeny/tabby/issues/6105#issuecomment-1242645726  @ouyangxudu @Hans-zhao831 @Fu1249519541 @zend

**Comment by lawweiliang** [comment]
@HiSunzhenliang 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
